test/Microsoft.AspNet.Mvc.ModelBinding.Test/Metadata/DefaultModelMetadataTest.cs

```diff
@@ -0,0 +1,451 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+#if !ASPNETCORE50
+using Moq;
+#endif
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Metadata
+{
+    public class DefaultModelMetadataTest
```

Comment:
please test `[Display]` providing `Description` and `Name` from resources (i.e. with a non-`null` `ResourceType` value) either here or in `DataAnnotationsMetadataDetailsProviderTest`.  those missing tests would have caught the earlier `GetName()` -> `Name` change